src/sfizz/Defaults.h

```diff
@@ -124,6 +124,48 @@ namespace Default
 	constexpr float rtDecay { 0.0f };
 	constexpr Range<float> rtDecayRange { 0.0f, 200.0f };
 
+    // Performance parameters: Filters
+    constexpr float numFilters { 2 };
+    constexpr float filterCutoff { 0 };
+    constexpr float filterResonance { 0 };
+    constexpr float filterGain { 0 };
+    constexpr int filterKeytrack { 0 };
+    constexpr uint8_t filterKeycenter { 60 };
+    constexpr int filterRandom { 60 };
+    constexpr int filterVeltrack { 0 };
+    constexpr int filterCutoffCC { 0 };
+    constexpr float filterResonanceCC { 0 };
+    constexpr float filterGainCC { 0 };
+    constexpr Range<float> filterCutoffRange { 0.0f, 20000.0f };
+    constexpr Range<int> filterCutoffModRange { -9600, 9600 };
+    constexpr Range<float> filterGainRange { -96.0f, 24.0f };
+    constexpr Range<float> filterGainModRange { -96.0f, 24.0f };
```

Comment:
This "peak/shelf gain" designates the level where the peak, or shelf, is located with relation to 0dB.
Not a proper a "gain" as such, just a level.

As such -96,+24 dB is not appropriate as range.
Possibly, peq is not expected to be used with a negative level, which makes it effectively a notch filter.
If negative levels are kept, at least they should be made symmetric around the 0dB. (eg ±24)

It's the same for shelfs, except instead of boosting a band, it will attenuate it in negatives.

As example. a "-20dB lowshelf"
![Capture du 2020-02-05 20-43-16](https://user-images.githubusercontent.com/17614485/73876964-3020cf80-4858-11ea-864a-f3c3ce247923.png)

vs a "+20dB" one
![Capture du 2020-02-05 20-43-55](https://user-images.githubusercontent.com/17614485/73877034-475fbd00-4858-11ea-9200-84a25a373be9.png)

Reply:
I will add ; +- 24dB is restrictive compared to what aria allows.
Based on some testing, it permits at least +- 80 dB of range.